src/components/member-card/card.module.scss

```diff
@@ -1,3 +1,21 @@
+.descriptionDivStyle {
+  background-color: rgb(0, 0, 0, 0.5);
+  position: absolute;
+  display: none;
+  color: white;
+  flex: 1 250px;
+  flex-direction: column;
+  border: none;
+  border-radius: 5%;
+  margin-top: -1.3rem;
```

Comment:
try to do margin in positive rem

Reply:
I tried giving positive bottom and right margins but it didn't worked for me

Reply:
Okay, let's not try to go for negative rems from next time.

Reply:
okay try it from next time next negative rems